core/src/com/unciv/ui/utils/KeyPressDispatcher.kt

```diff
@@ -66,9 +66,16 @@ data class KeyCharAndCode(val char: Char, val code: Int) {
         /** mini-factory for control codes - case insensitive */
         fun ctrl(letter: Char) = KeyCharAndCode(Char(letter.code and 31), 0)
```

Comment:
Why are we reducing all chars to only the first 5 bits? Is this just because we needed an arbitrary segment of chars that wouldn't collide with any other chars, or is there some deeper meaning?
In either case, I think a comment would be appropriate